documentation/modules/rn-28-resolved-issues.adoc

```diff
@@ -7,6 +7,18 @@
 
 {project-first} 2.8 has the following resolved issues:
 
+[id="resolved-issues-2-8-3_{context}"]
+== Resolved issues 2.8.3
+
+.NIC mapping for VMs with multiple NICs is unordered
```

Comment:
We are having problems with verifying this issue, so we might need to remove it from 2.8.3

Reply:
@mnecas OK, let me know.

Reply:
@mnecas I removed it as we discussed. Please review this PR again.